src/tests/test_control_flow/Makefile

```diff
@@ -0,0 +1,41 @@
+CC := clang
+LLC := ../../llvm/build/bin/llc
+CPOINTPASS := -Xclang -load -Xclang ../../passes/build/function_passes/libCheckPointPass.so
```

Comment:
Why don't you use a variable for the path prefix here? It would make things easier to read.